verticapy/tests_new/performance/vertica/conftest.py

```diff
@@ -0,0 +1,98 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import pytest
+import logging
+import os
+
+# verticapy is notebook integration
+import verticapy as vp
+
+# vertica_python is client connection
+import vertica_python
+
+
+@pytest.fixture()
+def vp_connect_get_name():
+    logging.info("Trying to connect")
+    info_map = get_connection_info()
+    connection_name = "pytest connection"
+    # TODO: when this fails, the error message has
+    # verbose stack information, but not instructions
+    # for how to fix things
+    vp.new_connection(info_map, name="pytest connection")
+    logging.info("Connection complete")
+    return connection_name
+
+
+@pytest.fixture
+def raw_client_cursor():
+    info_map = get_connection_info(verbose=False)
+    conn = None
+    try:
+        conn = vertica_python.connect(**info_map)
+        yield conn.cursor()
+    finally:
+        if conn is not None:
+            conn.close()
+
+
+def get_connection_info(verbose=True):
```

Comment:
@vikash018 do we need to recreate connection functions each time?

Reply:
@jslaunwhite-microfocus - can you try using current cursor? This is how currently we are doing if we need to run sql statement. 
from verticapy.connection import current_cursor
current_cursor().execute("select * from table" )

Reply:
I removed this and replaced it with asserts

`assert len(vp.available_connections()) > 0`

Reply:
I removed the connection setup and started using the connection setup recommended by @afard . I am able to get the desired behavior for my tests. It took some fussing with my environment to get things right, but now it is all set. You can see that the conftest.py file has been removed.